mesa/suite/bookworm/debian/patches/0001-PVR-changes.patch

```diff
@@ -1,42 +0,0 @@
-From: Andrew Davis <afd@ti.com>
-Date: Thu, 7 Sep 2023 19:17:28 -0500
-Subject: PVR changes
-
-Signed-off-by: Andrew Davis <afd@ti.com>
----
- src/gallium/targets/dri/meson.build | 7 ++++++-
- src/gallium/targets/dri/target.c    | 3 +++
- 2 files changed, 9 insertions(+), 1 deletion(-)
-
-diff --git a/src/gallium/targets/dri/meson.build b/src/gallium/targets/dri/meson.build
-index 1fd98bc..f2f1da2 100644
---- a/src/gallium/targets/dri/meson.build
-+++ b/src/gallium/targets/dri/meson.build
-@@ -116,7 +116,12 @@ foreach d : [[with_gallium_kmsro, [
-              [with_gallium_zink, 'zink_dri.so'],
-              [with_gallium_d3d12, 'd3d12_dri.so'],
-              [with_gallium_asahi, 'asahi_dri.so'],
--             [with_gallium_pvr, 'pvr_dri.so'],
-+             [with_gallium_pvr, [
-+               'pvr_dri.so',
-+               'tilcdc_dri.so',
-+               'omapdrm_dri.so',
-+               'tidss_dri.so',
```

Comment:
I believe we will still need this, even if we do not plan to support the SGX boards.

Reply:
yup. I've added them locally. Will push once after my build is done.